certified-connectors/tyntec-WA/apiDefinition.swagger.json

```diff
@@ -504,16 +508,2532 @@
+						  },
+						  "components": {
+							"type": "object",
+							"properties": {
+							  "body": {
+								"type": "array",
+								"items": {
+								  "type": "object",
+								  "properties": {
+									"type": {
+									  "type": "string",
+									  "description": "type"
+									},
+									"text": {
+									  "type": "string",
+									  "description": "text"
+									}
+								  }
+								},
+								"description": "body"
+							  },
+							  "button": {
+								"type": "array",
+								"items": {
+								  "type": "object",
+								  "properties": {
+									"type": {
+									  "type": "string",
+									  "description": "type"
+									},
+									"index": {
+									  "type": "integer",
+									  "format": "int32",
+									  "description": "index"
+									},
+									"payload": {
+									  "type": "string",
+									  "description": "payload"
+									}
+								  }
+								},
+								"description": "button"
+							  }
+							},
+							"description": "components"
+						  }
+						},
+						"description": "template"
+					  }
+					},
+					"description": "content",
+					"required": [
+					  "contentType"
+					]
+				  }
+				},
+				"required": [
+				  "channel",
+				  "content"
+				]
+			  }
+			}
+		  ],
+		  "operationId": "SendWhatsAppTemplateQuickReply",
+		  "summary": "Send WhatsApp Template Quick Reply",
+		  "description": "This action sends WhatsApp template with quick reply. Usable anytime. Requires registered template."
+		}
+	  },
+	  "/conversations/v3/power-automate/webhooks/channels/whatsapp/phone-numbers/{WABA}": {
+		"x-ms-notification-content": {
+		  "description": "Response",
+		  "schema": {
+			"type": "object",
+			"properties": {
+			  "messageId": {
+				"type": "string",
+				"description": "messageId"
+			  },
+			  "channel": {
+				"type": "string",
+				"description": "channel"
+			  },
+			  "from": {
+				"type": "string",
+				"description": "from"
+			  },
+			  "to": {
+				"type": "string",
+				"description": "to"
+			  },
+			  "receivedAt": {
+				"type": "string",
+				"description": "receivedAt"
+			  },
+			  "content": {
+				"type": "object",
+				"properties": {
+				  "contentType": {
+					"type": "string",
+					"description": "contentType"
+				  },
+				  "text": {
+					"type": "string",
+					"description": "text"
+				  }
+				},
+				"description": "content"
+			  },
+			  "event": {
+				"type": "string",
+				"description": "event"
+			  },
+			  "whatsapp": {
+				"type": "object",
+				"properties": {
+				  "senderName": {
+					"type": "string",
+					"description": "senderName"
+				  }
+				},
+				"description": "whatsapp"
+			  },
+			  "timestamp": {
+				"type": "string",
+				"description": "timestamp"
+			  }
+			}
+		  }
+		},
+		"post": {
+		  "responses": {},
+		  "summary": "Incoming message",
+		  "description": "Triggers on Incoming message",
+		  "operationId": "incoming_v3",
```

Comment:
@ridlees all operationIds should be in Camel case please change to "IncomingV3". please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Please fix all operationIds in your swagger file.

Reply:
Fixed, rest is CamelCase